srcpkgs/foliate/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'foliate'
+pkgname=foliate
+version=1.1.0
+revision=1
+build_style=meson
+configure_args="--prefix=/usr"
```

Comment:
This is already set by the build-style and can be removed.

Reply:
Thanks for the review. I removed that.